doc/en/user/source/security/layer.rst

```diff
@@ -14,16 +14,19 @@ Providing access to layers is linked to :ref:`roles <security_rolesystem_roles>`
 Rules
 -----
 
-The syntax for a layer security rule is as follows (``[]`` denotes optional
-parameters)::
+The syntax for a layer security rule can follow three different patterns (``[]`` denotes optional parameters)::
 
+  globalLayerGroup.permission=role[,role2,...]
   workspace.layer.permission=role[,role2,...]
+  workspace.workspaceLayerGroup.permission=role[,role2,...]
 
-The parameters include::
+The parameters include:
 
-* ``workspace``—Name of the workspace. The wildcard ``*`` is used to indicate all workspaces.
-* ``layer``—Name of a resource (featuretype/coverage/etc...). The wildcard ``*`` is used to indicate all layers.
-* ``permission``—Type of access permission/mode. 
+* ``globalLayerGroup``— Name of a global layer group.
+* ``workspace``— Name of the workspace. The wildcard ``*`` is used to indicate all workspaces.
+* ``layer``— Name of a resource (featuretype/coverage/etc...). The wildcard ``*`` is used to indicate all layers.
+* ``workspaceLayerGroup``— Name of a workspace specific layer group
+* ``permission``— Type of access permission/mode. 
    
    * ``r``—Read access
    * ``w``—Write access
```

Comment:
Nope, the code really does not make this consideration today, one has to remember to also state the read rule